core/projects/endpoints.go

```diff
@@ -129,10 +134,26 @@ func new(c echo.Context) error {
 		return c.NoContent(http.StatusBadRequest)
 	}
 
-	uuid := uuid.New().String()
+	projectPayload := form.Value["payload"]
+	if len(projectPayload) != 1 {
+		fmt.Println("more payloads than expected")
+		return c.NoContent(http.StatusBadRequest)
+	}
+
+	createProject := &CreateProject{}
+	err = json.Unmarshal([]byte(projectPayload[0]), createProject)
+	if err != nil {
+		fmt.Println(err)
+		return c.NoContent(http.StatusBadRequest)
+	}
 
-	path := fmt.Sprintf("%s/%s", runtime.Cfg.LibraryPath, uuid)
+	project := models.NewProject()
+	project.Name = createProject.Name
+	project.Path = "/"
```

Comment:
Reading the code I couldn't find a way where the Path is anything else than ´/´ when using the endpoint.
I may be wrong so let me know if that's not the case.

This way it seems easier to reason about rather than trying to do base rel paths and so on.